eslint.config.mjs

```diff
@@ -390,6 +390,18 @@ export default tseslint.config(
     },
     settings: { vitest: { typecheck: true } },
   },
+  {
+    files: ['packages/*/tests/**/vitest-custom-matchers.d.ts'],
```

Comment:
ACK that this comes from https://github.com/typescript-eslint/typescript-eslint/pull/11104/files#diff-afd33b740fd988dbbb18cbb1f285bd4032b350b2f8615153f217a55584b6aaba